sdk/runanywhere-react-native/packages/core/cpp/bridges/InitBridge.cpp

```
@@ -384,6 +814,29 @@ rac_result_t InitBridge::initialize(
+    // Use actual platform (ios/android) as backend only accepts these values
+#if defined(__APPLE__)
+    sdkConfig.platform = "ios";
+#elif defined(ANDROID) || defined(__ANDROID__)
+    sdkConfig.platform = "android";
+#else
+    sdkConfig.platform = "ios"; // Default to ios for unknown platforms
+#endif
+    sdkConfig.sdk_version = "0.2.0";  // TODO: Make configurable
+    sdkConfig.device_id = getPersistentDeviceUUID().c_str();
+    
+    rac_validation_result_t validResult = rac_sdk_init(&sdkConfig);
```

Cppcheck (2.19.0)

[error] Using object that is a temporary.

(danglingTemporaryLifetime)